app/models/activity.rb

```diff
@@ -53,6 +53,11 @@ def self.cs_accelerator_diagnostic_tool
     end
   end
 
+  def stem_course_template_no= value
+    # Ensure that all stem_course_template_no's are stored in downcase format
+    super(value.downcase)
+  end
+
```

Comment:
It's a shame we're not on 7.1 yet, we could have used the new normalize feature here https://www.bigbinary.com/blog/rails-7-1-adds-activerecord-base-normalizes

Reply:
@msquance-stem could you please use brackets around your setter's arguments?

`def stem_course_template_no=(value)`